dockerfiles/steps/git-assemble.bash

```diff
@@ -18,6 +20,11 @@ while read -r line; do # Loop over each <book> entry in the META-INF/books.xml m
 
     try cp "$path" "$IO_FETCH_META/modules/collection.xml"
 
+    if [[ -f temp-assembly/collection.assembled.xhtml ]]; then
+        rm temp-assembly/collection.assembled.xhtml # LCOV_EXCL_LINE
+    fi
+
+    export HACK_CNX_LOOSENESS=1 # Run neb assemble a bit looser. This deletes ToC items when the CNXML file is missing
```

Comment:
Should this be disabled?

Reply:
I fixed this one but not the `git-assemble-epub.bash` one because I'm hoping to release a version on DockerHub that we can use for all the GitHub Actions in cnx-user-books